main/utilities.py

```diff
@@ -8,7 +8,7 @@ def send_mail_notification(title, message, receiver_email):
 
 
 def send_site_notification(message, receiver_id):
-    """ notifiy the receiver id the message given using the site notification """
+    """ notify the receiver id the message given using the site notification """
```

Comment:
This sentence is not clear enough. Maybe you meant:
```suggestion
    """ notify the receiver by his id with the message given by the site notification """
```

Reply:
done. thanks 😄